js-core/homeworks/phonebook/index.js

```diff
@@ -0,0 +1,106 @@
+class PhoneApp {
+	constructor() {
+		this.dataBase = [
+        	{id:1, name:'Vasya', phone:'qweqwe'},
+        	{id:6, name:'Vasya', phone:'qweqwe'},
+        	{id:3, name:'Vasya', phone:'qweqwe'},
+        	{id:8, name:'Vasya', phone:'qweqwe'},
+        	{id: 10, name: 'Tom2', phone: '0995385'},
+        	{id: 10, name: 'Anita', phone: '0995305385'},
+        	{id: 12, name: 'Anita', phone: '9995305385'}
+    	]
+	};
+
+	checkForNumbers(phone) {
```

Comment:
probably in most cases there should be better name similar to

isPhoneNumberValid

meaning of "is" prefix at the beginning that function return boolean